libs/npress/AdminModule/templates/@layout.latte

```diff
@@ -12,7 +12,8 @@
 	<link rel="stylesheet" href="{$basePath}/static/bootstrap.css" type="text/css">
 	<link rel="stylesheet" href="{$basePath}/static/screen.css" type="text/css">
 	<link rel="shortcut icon" href="{$basePath}/favicon.ico" type="image/x-icon">
-	<script>document.write('<style>.jshidden { display:none }</style>')</script>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
```

Comment:
Trochu bych se bránil přidání celé nové knihovny ikonek, když už tam máme ikonky glyphicons. Viz https://bootstrapdocs.com/v2.0.4/docs/base-css.html#icons

Je fakt že 80 kB dneska nikoho nezabije. Asi to můžeš nechat, ale pokud je to možné obecně používej zdroje, které jsou v projektu již přidané.